tests/anumargak_test.js

```diff
@@ -282,47 +283,65 @@ describe("Anumargak ", function() {
         anumargak.on("GET", "/this/is/static/", () => 50);
         anumargak.on("GET", "/this/is/other/static", () => 70);
 
-        expect(anumargak.find("GET","/this/is/dynamic/with/123")() ).toEqual(30);
-        expect(anumargak.find("GET","/this/is/dynamic/with/123/")() ).toEqual(30);
-        expect(anumargak.find("GET","/this/is/static")() ).toEqual(50);
-        expect(anumargak.find("GET","/this/is/static/")() ).toEqual(50);
-        expect(anumargak.find("GET","/this/is/other/static")() ).toEqual(70);
-        expect(anumargak.find("GET","/this/is/other/static/")() ).toEqual(70);
+        expect(anumargak.find("GET", "/this/is/dynamic/with/123")()).toEqual(30);
+        expect(anumargak.find("GET", "/this/is/dynamic/with/123/")()).toEqual(30);
+        expect(anumargak.find("GET", "/this/is/static")()).toEqual(50);
+        expect(anumargak.find("GET", "/this/is/static/")()).toEqual(50);
+        expect(anumargak.find("GET", "/this/is/other/static")()).toEqual(70);
+        expect(anumargak.find("GET", "/this/is/other/static/")()).toEqual(70);
+    });
+
+    it("should ignore leading slash", function () {
+        var anumargak = Anumargak({
+            ignoreTrailingSlash: true,
+            ignoreLeadingSlash: true
+        });
+
+        anumargak.on("GET", "this/is/:dynamic/with/:pattern(\\d+)/", () => 30);
+        anumargak.on("GET", "this/is/static/", () => 50);
+        anumargak.on("GET", "this/is/other/static", () => 70);
+
```

Comment:
Please add `expect(anumargak.count).toEqual(3);` to ensure that all the routes are registered only once.

Reply:
It helps 👍 

Thank you @amitguptagwl